public/js/map/overlays/PlayerOverlay.js

```diff
@@ -26,55 +21,59 @@ export default L.LayerGroup.extend({
     this.onMinetestUpdate = this.onMinetestUpdate.bind(this);
   },
 
-  createPopup: function(player){
-    let html = "<b>" + player.name + "</b>";
-    html += "<hr>";
+  createPopup: function(player) {
+    // moderators get a small crown icon
+    let moderator = player.moderator ? `<img src="pics/crown.png">` : "";
 
-    for (let i=0; i<Math.floor(player.hp / 2); i++)
-      html += "<img src='pics/heart.png'>";
+    let info = `<b>${moderator} ${player.name}</b>`;
+    info += "<hr>";
 
-    if (player.hp % 2 == 1)
-      html += "<img src='pics/heart_half.png'>";
+    for (let i = 0; i < Math.floor(player.hp / 2); i++)
+      info += "<img src='pics/heart.png'>";
 
-    html += "<br>";
+    if (player.hp % 2 === 1)
+      info += "<img src='pics/heart_half.png'>";
 
-    for (let i=0; i<Math.floor(player.breath / 2); i++)
-      html += "<img src='pics/bubble.png'>";
+    info += "<br>";
 
-    if (player.breath % 2 == 1)
-      html += "<img src='pics/bubble_half.png'>";
+    for (let i = 0; i < Math.floor(player.breath / 2); i++)
+      info += "<img src='pics/bubble.png'>";
 
-    html += `
+    if (player.breath % 2 === 1)
+      info += "<img src='pics/bubble_half.png'>";
+
+    info += `
       <br>
       <b>RTT:</b> ${Math.floor(player.rtt*1000)} ms
       <br>
       <b>Protocol version:</b> ${player.protocol_version}
     `;
 
-    return html;
+    info = `<div class="info">${info}</div>`;
+
+    let portrait = `<img class="portrait" src="${this.getSkin(player)}" alt="${player.name}">`;
+
+    return `<div class="player-popup">${portrait}${info}</div>`;
   },
 
-  createMarker: function(player){
+  createMarker: function(player) {
     const marker = L.marker([player.pos.z, player.pos.x], {icon: this.getIcon(player)});
 
-    marker.bindPopup(this.createPopup(player));
+    marker.bindPopup(this.createPopup(player), {minWidth: 220});
```

Comment:
I made the popup wider by default just because it looked a bit squashed with the player portrait included.